create.ps1

```diff
@@ -413,7 +510,7 @@ try {
                     $outputContext.Data.Person = $personObject
                 }
```

Comment:
Message = "Multiple Persons found with [NAME = $($personAccount.Name)] AND [FIRSTNAME = $($personAccount.FirstName)], Please correct this so the Persons are unique."

And without a SalaryNumber ? Misschien nog goed om toe te voegen.